nix/stack-shell.nix

```diff
@@ -2,6 +2,13 @@ with import ./. { };
 
 haskell.lib.buildStackProject {
   name = "stack-env";
-  buildInputs = with pkgs; [ zlib openssl gmp libffi git haskellPackages.happy ];
+  buildInputs = with pkgs; [
+    zlib
```

Comment:
So for instance here we're using a multi-line packages specification. Maybe it's a good idea to stick to a single style.

Reply:
This formatting has been done automatically by `nixfmt`.